src/Tribe/Views/V2/Assets.php

```diff
@@ -99,6 +99,49 @@ public function register() {
 			]
 		);
 
+		/**
+		 * @todo: @paulmskim figure out how to enqueue only when widget is available.
+		 * @todo: @paulmskim commented out for release so widget styles do not enqueue, uncomment when actually releasing.
+		 */
+		// tribe_asset(
```

Comment:
:warning: This comment is 50% valid code; is this commented out code?

<sub>hash: 27f994313dad8231b64be0067d90d4a8</sub>